src/components/ConfirmRequestDialog.tsx

```diff
@@ -38,25 +39,39 @@ const ConfirmRequestDialog: FunctionComponent<Props> = ({
         <br />
         <strong>{values.reason}</strong>
       </p>
-      <p className="lbh-body">{formatSentence(deliveryMethods)}</p>
 
-      <ul className="lbh-list">
+      <p className="lbh-body">{formatSentence(deliveryMethods)}</p>
+      <ul className="lbh-list govuk-!-margin-top-2">
         <li>
           <strong>{values.resident.name}</strong>
         </li>
-        <li>{values.resident.email}</li>
-        <li>{values.resident.phoneNumber}</li>
+        <li>
+          <strong>{values.resident.email}</strong>
+        </li>
+        <li>
+          <strong>{values.resident.phoneNumber}</strong>
+        </li>
       </ul>
 
       <p className="lbh-body">For the following evidences:</p>
-      <ul className="lbh-list lbh-list--bullet">
+      <ul className="lbh-list lbh-list--bullet govuk-!-margin-top-2">
         <strong>
           {values.documentTypes.map((id) => (
             <li key={id}>{documentTypes.find((dt) => dt.id == id)?.title}</li>
           ))}
         </strong>
       </ul>
 
+      {values.noteToResident.trim() && (
```

Comment:
We are now trimming in two separate places in the code, i.e. here and before submitting to `evidence-api`. Perhaps this sanitisation logic could be in a single place.

Reply:
Refactored this to include a new sanitiseNoteToResident method in the helpers directory